app/javascript/react/components/Map/Markers/Markers.tsx

```diff
@@ -42,15 +48,30 @@ const Markers = ({ points }: Props) => {
     });
   };
 
+  const centerMapOnMarker = (position: LatLngLiteral, key: string) => {
+    if (map) {
+      map.setCenter(position);
+      map.setZoom(ZOOM_FOR_SELECTED_SESSION);
+    }
+    setSelectedMarkerKey(key === selectedMarkerKey ? null : key);
+  };
+
   return (
     <>
-      {points.map((point) => (
+      {sessions.map((session) => (
         <AdvancedMarker
-          position={point}
-          key={point.key}
-          ref={(marker) => setMarkerRef(marker, point.key)}
+          position={session.point}
+          key={session.point.key}
+          ref={(marker) => setMarkerRef(marker, session.point.key)}
         >
-          <span className="tree">🌳</span>
+          <SingleMarker
+            color="#E95F5F"
+            value={`${session.lastMeasurementValue} µg/m³`}
```

Comment:
Is it always will be this unit - `µg/m³`? Or do we need to take this unit data from the backend?

Reply:
It's not, it depends on the filter's measurement type, it can be Fahrenheit for example. But tbh I'm not sure where we get those, as we have `  const unit_symbol = "µg/m³";
` in filters that we send to the backend. It would mean that we should know the unit symbol.  🤔

Soooo any ideas on how to handle it for now?

Reply:
I would work on this while working on the filters, for now, I'll just add a comment and leave a note in the appropriate ticket to figure it out later